### ISSUE TITLE
The use of pi (mathematicalConstant::pi) should be Foam::constant::mathematical::pi

### ISSUE THREAD

**Issue by Daniel-UCAS** [wyldckat/perturbU#1]
In the code:
`   

```
const scalar betaPlus = 2.0*mathematicalConstant::pi*(1.0/200.0);

const scalar sigma = 0.00055;

//streamwise wave number: spacing x+ = 500

const scalar alphaPlus = 2.0*mathematicalConstant::pi*(1.0/500.0);
```

`

the use of pi is not correct, see the definition in $FOAM_SRC/OpenFOAM/global/constants/mathematical/mathematicalConstants.H

**Comment by timofeymukha** [comment]
Confirmed for version 3.0.1.

**Comment by wyldckat** [comment]
@Danniel-UCAS: Sorry, I forgot to make a note on my to-do list to take a look into this.

@Danniel-UCAS  and @timofeymukha: Nonetheless, something very strange is going on here, because:
1. To which branch are you two referring to?
2. To which file are you two referring to?
3. Did you follow the instructions given at the wiki page: http://openfoamwiki.net/index.php/Contrib/perturbU ??

**Comment by wyldckat** [comment]
OK, I've finally managed to give a bit more attention to this and it looks like you two didn't notice that the reported issue is already fixed in the branch [OF20x](/wyldckat/perturbU/tree/OF20x) and [OF30x](/wyldckat/perturbU/tree/OF30x), which is documented on the wiki page that these two are the correct branches for OpenFOAM 2.x and 3.x.

The reported issue was fixed on the 21st of March 2015, in commit  f7727f365da23ad20038a5878ecde0007a6f3213

Anyway, to force people to take a better look at the branches and the wiki, I've renamed the old master branch to [OF17x](/wyldckat/perturbU/tree/OF17x) and created a new [master](/wyldckat/perturbU) branch that acts as a front-page. Hopefully this will avoid future problems.

I'm closing this issue as already fixed.
